accumulo-store/src/main/java/gaffer/accumulostore/utils/AccumuloStoreConstants.java

```diff
@@ -25,7 +25,8 @@
     public static final String BLOOM_FILTER_ITERATOR_NAME = "Bloom_Filter";
     public static final String ELEMENT_FILTER_ITERATOR_NAME = "Element_Filter";
     public static final String EDGE_ENTITY_DIRECTED_UNDIRECTED_INCOMING_OUTGOING_FILTER_ITERATOR_NAME = "Edge_Entity_Directed_Undirected_Incoming_Outgoing_Filter";
-    public static final String QUERY_TIME_AGGREGATION_ITERATOR_NAME = "Query_Time_Aggregator";
+    public static final String COLUMN_QUALIFIER_AGGREGATOR_ITERATOR_NAME = "Column_Qualifier_Aggregator";
```

Comment:
That will aggregate the column qualifier downwards so yes including visibility timestamp and value
I thought when compared with the RowID aggregator it'd start to make more contextual sense as the iterator naming pattern would be that key portion down.